src/Ledger/Foreign/LedgerTypes.agda

```diff
@@ -81,7 +81,7 @@ record PParams : Set where
         maxTxSize        : ℕ
         maxHeaderSize    : ℕ
         maxValSize       : ℕ
-        minUTxOValue     : Coin
+        minUtxOValue     : Coin
```

Comment:
I think, for consistency, we are changing all appearances of "UTxO" in a variable name to Utxo (but I may be wrong about this).